draft-ietf-quic-recovery.md

```diff
@@ -1132,6 +1144,8 @@ increases bytes_in_flight.
 
 ~~~
    OnPacketSentCC(bytes_sent):
+     if (bytes_in_flight == 0):
+       probe_start_time = Now()
```

Comment:
It's important not to start this if the packet isn't ack-eliciting, but I will say that requirement makes this more complex, since you have to track the first ack-eliciting packet to be sent.

Reply:
So, are you suggestion a change? Perhaps a comment?